Origami/src/components/App.js

```diff
@@ -24,8 +24,10 @@ class App extends React.Component {
       login: false,
       displayLogin: "",
       showTitle: true,
-      isFrame: window.location.pathname.split("/")[1] == "frame"
+      isFrame: window.location.pathname.split("/")[1] == "frame",
+      isRoot:false
```

Comment:
Instead of using states here, don't allow anyone else apart from the root user to access the URL

Reply:
ohk ill do the necessary changes.